src/site/content/en/blog/streams/index.md

```diff
@@ -0,0 +1,966 @@
+calling its constructor
+[`WritableStream()`](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream/WritableStream).
+It has an optional argument called the stream's `underlyingSink`, which represents an object
+containing methods and properties that define how the constructed stream instance will behave. The
+`underlyingSink` can contain the following optional, developer-defined methods: The `controller`
+parameter passed to some of the following methods is a
+[`WritableStreamDefaultController`](https://developer.mozilla.org/en-US/docs/Web/API/WritableStreamDefaultController).
+
+#### The `underlyingSink`
+
+- `start(controller)`: This method is called immediately when the object is constructed. The
+  contents of this method should aim to get access to the underlying sink. If this process is to be
+  done asynchronously, it can return a promise to signal success or failure.
+- `write(chunk, controller)`: This method will be called when a new chunk of data (specified in the
+  `chunk` parameter) is ready to be written to the underlying sink. It can return a promise to
+  signal success or failure of the write operation. This method will be called only after previous
+  writes have succeeded, and never after the stream is closed or aborted.
+- `close(controller)`: This method will be called if the app signals that it has finished writing
+  chunks to the stream. The contents should do whatever is necessary to finalize writes to the
+  underlying sink, and release access to it. If this process is asynchronous, it can return a
+  promise to signal success or failure. This method will be called only after all queued-up writes
+  have succeeded.
+- `abort(reason)`: This method will be called if the app signals that it wishes to abruptly close
+  the stream and put it in an errored state. It can clean up any held resources, much like
+  `close()`, but `abort()` will be called even if writes are queued up—those chunks will be thrown
+  away. If this process is asynchronous, it can return a promise to signal success or failure. The
+  `reason` parameter contains a `DOMString` describing why the stream was aborted.
+
+The
+[`WritableStreamDefaultController`](https://developer.mozilla.org/en-US/docs/Web/API/WritableStreamDefaultController)
+interface of the Streams API represents a controller allowing control of a `WritableStream`'s state
+during set up, as more chunks are submitted for writing, or at the end of writing. When constructing
+a `WritableStream`, the underlying sink is given a corresponding `WritableStreamDefaultController`
+instance to manipulate. The `WritableStreamDefaultController` has only one method:
+[`WritableStreamDefaultController.error()`](https://developer.mozilla.org/en-US/docs/Web/API/WritableStreamDefaultController/error),
+which causes any future interactions with the associated stream to error.
+
+#### The `queuingStrategy`
```

Comment:
See comments from the previous `queuingStrategy` section

Reply:
Fixed throughout.